Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedCursorAndChangeStreamOperations.swift

```diff
@@ -68,10 +68,38 @@ struct IterateUntilDocumentOrError: UnifiedOperationProtocol {
     static var knownArguments: Set<String> { [] }
 
     func execute(on object: UnifiedOperation.Object, context: Context) throws -> UnifiedOperationResult {
-        let cs = try context.entities.getEntity(from: object).asChangeStream()
-        guard let next = cs.next() else {
-            throw TestError(message: "Change stream unexpectedly exhausted")
+        let entity = try context.entities.getEntity(from: object)
+        switch entity {
```

Comment:
this operation now supports multiple target types: https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#iterateuntildocumentorerror

accordingly, I renamed this file since the operations aren't just change stream specific anymore